docs/project/list-of-diagnostics.md

```diff
@@ -70,6 +70,7 @@ The PR that reveals the implementation of the `<IncludeInternalObsoleteAttribute
 |  __`SYSLIB0018`__ | ReflectionOnly loading is not supported and throws PlatformNotSupportedException. |
 |  __`SYSLIB0019`__ | RuntimeEnvironment members SystemConfigurationFile, GetRuntimeInterfaceAsIntPtr, and GetRuntimeInterfaceAsObject are no longer supported and throw PlatformNotSupportedException. |
 |  __`SYSLIB0020`__ | JsonSerializerOptions.IgnoreNullValues is obsolete. To ignore null values when serializing, set DefaultIgnoreCondition to JsonIgnoreCondition.WhenWritingNull. |
+|  __`SYSLIB0023`__ | RNGCryptoServiceProvider is obsolete. Use RandomNumberGenerator.Create() instead. |
```

Comment:
I think the actual guidance we want is to completely abandon the instances and just use the static RandomNumberGenerator.Fill method.

Would you like to assent or rebut, @GrabYourPitchforks?

Reply:
I agree with Jeremy. The only people who see this message are devs targeting .NET 6, and we should direct all such users to the static methods on `RandomNumberGenerator` instead.

There is one exception: if you're using DI to inject an instance of `RandomNumberGenerator` into your testable classes, you'll want to use `RandomNumberGenerator.Create()` to create an instance and register it as your singleton.

Reply:
Thanks. Updated; let me know if you have better wording though.